app/packs/src/apps/mydb/inbox/UnsortedDatasetModal.js

```diff
@@ -7,38 +8,34 @@ import Container from 'src/models/Container';
 export default class UnsortedDatasetModal extends React.Component {
   render() {
     const { show, datasetContainer, onHide } = this.props;
-    return (
-      show && (
-          <Modal centered animation show={show} size="lg" onHide={() => onHide()}>
+    if (show) {
+      return (
+        <div>
+          <Modal
+            centered
+            animation
+            show={show}
+            size="lg"
+            onHide={() => onHide()}
+          >
             <Modal.Header closeButton>
               <Modal.Title>
                 Upload files to Inbox
               </Modal.Title>
             </Modal.Header>
-            <Modal.Body className="mt-3">
+            <Modal.Body>
               <UnsortedDataset
                 datasetContainer={datasetContainer}
                 onModalHide={() => onHide()}
               />
             </Modal.Body>
-            <Modal.Footer className="border-0" >
-              <ButtonToolbar className="gap-1">
-              <Button
-                variant="primary"
-                onClick={() => onHide()}
-              >
-                Close
-              </Button>
-                <Button
-                  variant="warning"
-                  onClick={() => this.handleSave()}
-                >
-                  Save
-                </Button>
-              </ButtonToolbar>
-            </Modal.Footer>
           </Modal>
-      ));}
+        </div>
+      )
+    } else {
+      return null;
```

Comment:
NIT: I prefer an early return when `!show` but that is a personal preference.

Reply:
Yes, looks a lot cleaner, made the changes